blocks/donor-wall/edit/inspector.js

```diff
@@ -2,145 +2,253 @@
-					label={ __( 'Show Donation Form', 'give' ) }
-					checked={ !! showForm }
-					onChange={ ( value ) => saveSetting( 'showForm', value ) } />
-				<ToggleControl
-					name="showTotal"
-					label={ __( 'Show Total', 'give' ) }
-					checked={ !! showTotal }
-					onChange={ ( value ) => saveSetting( 'showTotal', value ) } />
-				<ToggleControl
-					name="showDate"
-					label={ __( 'Show Time', 'give' ) }
-					checked={ !! showDate }
-					onChange={ ( value ) => saveSetting( 'showDate', value ) } />
-				<ToggleControl
-					name="showComments"
-					label={ __( 'Show Comments', 'give' ) }
-					checked={ !! showComments }
-					onChange={ ( value ) => saveSetting( 'showComments', value ) } />
-				<ToggleControl
-					name="showAnonymous"
-					label={ __( 'Show Anonymous', 'give' ) }
-					checked={ !! showAnonymous }
-					onChange={ ( value ) => saveSetting( 'showAnonymous', value ) } />
-				<ToggleControl
-					name="onlyComments"
-					label={ __( 'Only Donors with Comments', 'give' ) }
-					checked={ !! onlyComments }
-					onChange={ ( value ) => saveSetting( 'onlyComments', value ) } />
-				<TextControl
-					name="commentLength"
-					label={ __( 'Comment Length', 'give' ) }
-					value={ commentLength }
-					onChange={ ( value ) => saveSetting( 'commentLength', value ) } />
-				<TextControl
-					name="readMoreText"
-					label={ __( 'Read More Text', 'give' ) }
-					value={ readMoreText }
-					onChange={ ( value ) => saveSetting( 'readMoreText', value ) } />
-				<TextControl
-					name="loadMoreText"
-					label={ __( 'Load More Text', 'give' ) }
-					value={ loadMoreText }
-					onChange={ ( value ) => saveSetting( 'loadMoreText', value ) } />
-			</PanelBody>
-		</InspectorControls>
+                <Panel>
+                    <PanelBody title= {__('Layout', 'give')} initialOpen={ true }>
+                        <ColumnSelector
+                            selected={columns}
+                            onClick={(value) => saveSetting('columns', value)}
+                            help={__('Controls how many columns of the Form Grid appear. All sizes will adjust responsively to the space available. The maximum number allowed per row is 4', 'give')}
+                        />
+                        <SelectControl
+                            className="give-donor-wall-inspector"
+                            name="columns"
+                            label={ __( 'Columns', 'give' ) }
+                            value={ columns }
+                            options={ giveDonorWallOptions.columns }
+                            onChange={ ( value ) => saveSetting( 'columns', value ) } />
+                    </PanelBody>
+                </Panel>
+                <Panel>
+                    <PanelBody title= {__('Display Elements', 'give')} initialOpen={ false }>
+                        <ToggleOptions
+                            options={[__( 'Donor info', 'give' ), __( 'Wall attributes', 'give' ) ]}
```

Comment:
@JoshuaHungDinh This will not work as expected because we are comparing translated labels against non translated strings.